src/optionDefaults.js

```diff
@@ -111,5 +111,10 @@ export const optionDefaults = {
       showControls: false,
       hidden: false,
     },
+    number: {
+      title: "Numbers",
```

Comment:
Is this the group-name you want?
I don't have numeric controls myself, but I wonder if "Numbers" is self-explanatory enough for the entities listed under it.
If you say it's OK, it's OK. It can always be changed when we find a better description.

Reply:
Yeah it's not perfect, but it's what home assistant calls it so I guess it's fine.